src/CreatedTask.js

```diff
@@ -1,10 +1,16 @@
 import React from "react"
 
+import './CreatedTask.css';
+
 export default function CreatedTask(props) {
   return (
     <div>
-      <input type="checkbox" checked={props.completed}/>
-      <p>{props.task}</p>
+      <input 
+        type="checkbox"
+        checked={props.completed}
+        onChange={() => props.handleClick(props.id)}
+      />
+      <p class="strikethrough">{props.task}</p>
```

Comment:
Have you seen the [`del`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/del) element?

Reply:
No!  Looking it up now :)